app/assets/javascripts/alchemy/templates/page_folder.hbs

```diff
@@ -0,0 +1,3 @@
+<a class="page_folder" data-page-id="{{ page.id }}">
```

Comment:
Hm, this is required in `package/src` now, should it go into the JS package instead of in the sprockets bundle?